packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -122,36 +123,52 @@ class MethodWebsocketRequestHandler {
     WebSocket webSocket,
     OpenMethodStreamCommand message,
   ) async {
-    // Validate targeted endpoint method
-    var endpointConnector =
-        server.endpoints.getConnectorByName(message.endpoint);
-    if (endpointConnector == null) {
+    late MethodStreamSession session;
```

Comment:
Suggestion: This session must be closed if we fail to call the streaming endpoint.

Reply:
That's what I reckoned as well, but this is actually not the current behavior, and _if_ I do that then a bunch of tests starts failing. Since this is the current behavior I just kept it the same, but maybe there's something here that needs to change. Lets discuss together! 👍

Reply:
[640023c](https://github.com/serverpod/serverpod/pull/2676/commits/640023c5e4997b5caa1f3ec9a420324d63e493db)